tests/common.py

```diff
@@ -175,6 +176,11 @@ def reset_logging():
     def account_id(self):
         return ACCOUNT_ID
 
+    @property
+    def current_user_arn(self):
```

Comment:
afaics this isn't used anywhere, also fwiw, this is equivalent to sts get caller identity, which we have in utils.py

Reply:
@kapilt - you are right that it isn't currently used.  I added it when I thought we were going to have to put the current user into some of the policies.  @whit537 was looking to see if he could remove all the users from those policies, but I had already written this helped function so I included it.